.github/workflows/publish-prod.yml

```diff
@@ -64,20 +54,17 @@ jobs:
       - name: Run testing
         run: make test
 
-      - name: Create docker image
-        run: make docker
-
-      - name: Publish docker image
-        run: |
-          docker tag ktranslate:v2 kentik/ktranslate:v2
-          docker push kentik/ktranslate:v2
-          docker tag ktranslate:v2 kentik/ktranslate:${{ env.KENTIK_KTRANSLATE_VERSION }}
-          docker push kentik/ktranslate:${{ env.KENTIK_KTRANSLATE_VERSION }}
-          docker tag ktranslate:v2 kentik/ktranslate:latest
-          docker push kentik/ktranslate:latest
+      - name: Build and Publish Docker
```

Comment:
@i3149 If this is OK, we will need to add a `build-arg` to use the `MAXMIND_LICENSE_KEY` env var in the Dockerfile.

Reply:
Makes sense to me.